theano/tensor/var.py

```diff
@@ -355,7 +355,7 @@ def astype(self, dtype):
 
     # SLICING/INDEXING
     def __getitem__(self, args):
-        if not isinstance(args, tuple):
+        if not isinstance(args, (list, tuple)):
             args = args,
```

Comment:
In fact, I think the only case where we should treat a list like a tuple is if the list has slices in it.